app/src/main/java/edu/uc/hickmadc/customnotifications/MainActivity.kt

```diff
@@ -1,38 +1,34 @@
 package edu.uc.hickmadc.customnotifications
 
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.widget.ListView
+import androidx.appcompat.app.AppCompatActivity
 import androidx.navigation.Navigation
 import androidx.navigation.findNavController
 import androidx.navigation.ui.NavigationUI
 import com.google.android.material.bottomnavigation.BottomNavigationView
 import edu.uc.hickmadc.customnotifications.ui.main.MyListAdapter
 
-//import edu.uc.hickmadc.customnotifications.ui.main.MainFragment
-
 class MainActivity : AppCompatActivity() {
 
```

Comment:
Another technical suggestion here which goes hand-in-hand with the one below this would be to implement unit tests here to see if the list population persists.